com.slugchess.SlugChess.yml

```diff
@@ -0,0 +1,59 @@
+app-id: com.slugchess.SlugChess
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: '/app/bin/slugchess-run.sh'
+sdk-extensions: 
+  - org.freedesktop.Sdk.Extension.dotnet7
+build-options: 
+  append-path: '/usr/lib/sdk/dotnet7/bin'
+  append-ld-library-path: '/usr/lib/sdk/dotnet7/lib'
+  env: 
+    PKG_CONFIG_PATH: '/app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig:/usr/lib/sdk/dotnet7/lib/pkgconfig'
+    DOTNET_CLI_TELEMETRY_OPTOUT: 'true'
+  arch:
+#    arm: 
+#      env: 
+#        RUNTIME: linux-arm
+#    aarch64: 
+#      env: 
+#        RUNTIME: linux-arm64
+    x86_64: 
+      env: 
+        RUNTIME: linux-x64
+modules:
+  - name: flatpak-dotnet
```

Comment:
this would be better if it was named 'slugchess', purealy from a readability standpoint.

Reply:
I have committed this change